docs/src/user-docs/quick-start/clp-text.md

```diff
@@ -29,6 +29,19 @@ If CLP fails to start (e.g., due to a port conflict), try adjusting the settings
 `etc/clp-config.yml` and then run the start command again.
 :::
 
+:::{warning}
+**Do not comment out or remove the `package` block in `etc/clp-config.yml`**, as the default storage
+engine is `clp-s`, which is optimized for JSON logs rather than unstructured text logs.
```

Comment:
```suggestion
**Do not comment out or remove the `package` block in `etc/clp-config.yml`**; otherwise, the storage
and query engines will default to `clp-s`, which is optimized for JSON logs rather than unstructured
text logs.
```